File: app/src/main/java/com/gigaworks/tech/bible/AboutActivity.java
package com.gigaworks.tech.bible;

import android.content.Context;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

import java.text.SimpleDateFormat;
import java.util.Date;

public class AboutActivity extends AppCompatActivity {

    private TextView version, build;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);

        version = findViewById(R.id.version);
        build = findViewById(R.id.build);

        Date buildDate = BuildConfig.buildTime;
        SimpleDateFormat sdf = new SimpleDateFormat("dd MMM yyyy");
        build.setText("Build Date: " + sdf.format(buildDate));
        version.setText("Version: " + BuildConfig.VERSION_NAME);

    }
}

File: app/src/main/java/com/gigaworks/tech/bible/ui/about/AboutActivity.kt
package com.gigaworks.tech.bible.ui.about

import android.os.Bundle
import android.view.LayoutInflater
import com.gigaworks.tech.bible.BuildConfig
import com.gigaworks.tech.bible.databinding.ActivityAboutBinding
import com.gigaworks.tech.bible.ui.base.BaseActivity
import java.text.SimpleDateFormat
import java.util.*

class AboutActivity : BaseActivity<ActivityAboutBinding>() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setupActionBar(binding.toolbar, "About") {
            finish()
        }

        val buildDate = BuildConfig.buildTime
        val sdf = SimpleDateFormat("dd MMM yyyy", Locale.getDefault())
        val buildDateText = "Build Date: " + sdf.format(buildDate)
        val versionText = "Version: " + BuildConfig.VERSION_NAME

        binding.build.text = buildDateText
        binding.version.text = versionText
    }

    override fun getViewBinding(inflater: LayoutInflater) = ActivityAboutBinding.inflate(inflater)
}